python/pyiceberg/cli/console.py

```diff
@@ -0,0 +1,185 @@
+
+from pyiceberg.catalog.rest import RestCatalog
+from pyiceberg.exceptions import NamespaceNotEmptyError, NoSuchNamespaceError, NoSuchTableError
+
+
+@click.group()
+@click.option("--catalog", type=click.Choice(["rest"], case_sensitive=False), default="rest")
+@click.option("--uri", default=lambda: os.environ.get("PYICEBERG_URI"))
+@click.option("--credentials", default=lambda: os.environ.get("PYICEBERG_CREDENTIALS"))
+@click.pass_context
+def run(ctx, catalog, uri, credentials):
+    if catalog == "rest":
+        ctx.ensure_object(dict)
+        ctx.obj["catalog"] = RestCatalog(name="catalog", properties={}, uri=uri, credentials=credentials)
+
+
+@run.command()
+@click.pass_context
+def list_namespace(ctx):
+    # Add parent namespaces
+    catalog: RestCatalog = ctx.obj["catalog"]
+    identifiers = catalog.list_namespaces()
+
+    table = Table(title="Namespaces")
+
+    table.add_column("Namespace")
+
+    for identifier in identifiers:
+        table.add_row(".".join(identifier))
+
+    console = Console()
+    console.print(table)
+
+
+@run.command()
+@click.argument("namespace")
+@click.pass_context
+def load_namespace(ctx, namespace: str):
+    catalog: RestCatalog = ctx.obj["catalog"]
+    console = Console()
+
+    try:
+        properties = catalog.load_namespace_properties(namespace)
+
+        if properties:
+            table = Table(title=f"{namespace} properties", show_header=False)
+            for key, value in properties.items():
+                table.add_row(key, value)
+            console.print(table)
+        else:
+            console.print("No properties...")
+    except NoSuchNamespaceError:
+        console.print(f"Namespace does not exists: {namespace}")
+
+
+@run.command()
+@click.argument("namespace")
+@click.pass_context
+def drop_namespace(ctx, namespace: str):
+    catalog: RestCatalog = ctx.obj["catalog"]
+    console = Console()
+
+    try:
+        catalog.drop_namespace(namespace)
+        console.print(f"Namespace {namespace} has been dropped")
+    except NoSuchNamespaceError:
+        console.print(f"Namespace does not exists: {namespace}")
+    except NamespaceNotEmptyError:
+        console.print(f"Namespace is not empty: {namespace}")
+
+
+@run.command()
+@click.argument("namespace")
+@click.pass_context
+def list_tables(ctx, namespace: str):
+    catalog: RestCatalog = ctx.obj["catalog"]
+    console = Console()
+    try:
+        identifiers = catalog.list_tables(namespace)
+
+        table = Table(title="Tables")
+
+        table.add_column("Table name")
+
+        for identifier in identifiers:
+            table.add_row(".".join(identifier))
+
+        console.print(table)
+    except NoSuchNamespaceError:
+        console.print(f"Namespace does not exists: {namespace}")
+
+
+@run.command()
+@click.argument("table")
+@click.pass_context
+def load_table(ctx, table: str):
+    catalog: RestCatalog = ctx.obj["catalog"]
+    console = Console()
+
+    try:
+        tbl = catalog.load_table(table)
```

Comment:
If we move to using `grid`, it's probably okay to continue using table here, since this is a bunch of different information mixed together.